CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning (http://semver.org/).
 
+## 4.0.7
+* Add a null check for customer before trying to populate customer data in order confirmation API call
```

Comment:
API call can be a little bit misleading, maybe using graphql keyword may be better.